airbyte-cdk/bulk/toolkits/extract-jdbc/src/main/kotlin/io/airbyte/cdk/read/JdbcPartitionReader.kt

```diff
@@ -150,6 +150,7 @@ open class JdbcResumablePartitionReader<P : JdbcSplittablePartition<*>>(
                 streamState.updateLimitState { it.down }
             }
         }
+        println("last record: $lastRecord")
```

Comment:
```suggestion
```

Reply:
sorry about these litering -  thanks for catching them

Reply:
that's what reviews are for! if you don't already, I suggest you do a self-review by looking at the diff on github after opening a PR